probe-rs/src/architecture/arm/memory/adi_v5_memory_interface.rs

```diff
@@ -591,7 +591,7 @@ mod tests {
         for address in 0..8 {
             let value = mi
                 .read_word_8(address)
-                .expect(&format!("read_word_8 failed, address = {}", address));
+                .unwrap_or_else(|_| panic!("read_word_8 failed, address = {}", address));
```

Comment:
is this the same as `expect`?

Reply:
ok the reason is to defer the format string to only time of failure, not to get rid of panic entirely.

since the panic is inside a test, it is ok